test-e2e/tests/live_channel_test.ts

```diff
@@ -0,0 +1,237 @@
+import assert from 'assert';
+
+import constants from '../utils/constants';
+
+const selectedProgramBackgroundColor = 'rgb(204, 204, 204)';
+const liveProgramOutline = 'rgb(255, 255, 255) solid 2px';
+
+const programBackgroundColor = 'rgba(255, 255, 255, 0.08)';
+const programOutline = 'rgb(255, 255, 255) none 0px';
+
+const channel1Id = 'Uh7zcqVm';
+const channel2Id = 'Z2evecey';
+
+const channel1LiveProgramId = 'f99402a4-783d-4298-9ca2-d392db317927';
+const channel1PreviousProgramId = 'a2f80690-86e6-46e7-8a78-b243d5c8237e';
+const channel1UpcomingProgram = '6ffd595b-3cdb-431b-b444-934a0e4f63dc';
+const channel2LiveProgramId = 'b628784b-1061-4a61-98f3-fa48ce3ce81f';
+
+Feature('live channel').retry(3).tag('@desktop-only');
+
+Before(({ I }) => {
+  I.useConfig('test--blender');
+});
+
+const videoDetailLocator = locate({ css: 'div[data-testid="video-detail"]' });
+
+const shelfContainerLocator = locate({ css: 'div[role="row"]' });
+const shelfLocator = locate({ css: 'div[role="cell"]' }).inside(shelfContainerLocator);
+const epgContainer = locate({ css: 'div[data-testid="container"]' });
+const epgProgram = (id: string) => locate({ css: `div[data-testid="${id}"]` }).inside(epgContainer);
+const epgChannel = (id: string) => locate({ css: `div[data-testid="${id}"]` }).inside(epgContainer);
```

Comment:
Make the difference clear in naming between static locators and creator functions:

```suggestion
const shelfContainerLocator = locate({ css: 'div[role="row"]' });
const shelfLocator = locate({ css: 'div[role="cell"]' }).inside(shelfContainerLocator);
const epgContainerLocator = locate({ css: 'div[data-testid="container"]' });
const makeEpgProgramLocator = (id: string) => locate({ css: `div[data-testid="${id}"]` }).inside(epgContainer);
const makeEpgChannelLocator = (id: string) => locate({ css: `div[data-testid="${id}"]` }).inside(epgContainer);
```